source/renderer/app/components/status/DaedalusDiagnostics.scss

```diff
@@ -331,9 +300,17 @@ td.stateDirectoryPath {
     transition: 0.15s opacity;
     width: 11px;
   }
+
+  @media (min-width: 1001px) and (max-width: 1560px) {
```

Comment:
@DeeJayElly can we get rid of this media query?

Reply:
We need this max-width calculation only in the case where the text is breaking. I can remove it but then i am trying to force some width calculations on the resolutions where i don't need calculations at all.